apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/dao/DexOrderDao.java

```diff
@@ -34,7 +34,7 @@
             "AND (:offerCur is NULL OR offer.offer_currency = :offerCur) " +
             "AND (:pairCur is NULL OR offer.pair_currency = :pairCur) " +
             "ORDER BY offer.pair_rate DESC " +
-            "OFFSET :offset LIMIT :limit"
+            "OFFSET :offset FETCH FIRST :limit ROWS ONLY"
```

Comment:
Standard limit is not working? In this doc limit can be used `http://www.h2database.com/html/commands.html#select`

Reply:
Even though LIMIT is supposed to work, this "OFFSET :offset LIMIT :limit" leads to 
`Caused by: org.jdbi.v3.core.statement.UnableToCreateStatementException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM DEX_OFFER AS OFFER WHERE LATEST = TRUE AND (OFFER.DB_ID > ?) AND (? IS NULL OR OFFER.ACCOUNT_ID = ?) AND (? IS NULL OR OFFER.FINISH_TIME > ?) AND (? IS NULL OR OFFER.TYPE = ?) AND (? IS NULL OR OFFER.STATUS = ?) AND (? IS NULL OR OFFER.OFFER_CURRENCY = ?) AND (? IS NULL OR OFFER.PAIR_CURRENCY = ?) ORDER BY OFFER.PAIR_RATE DESC OFFSET ? LIMIT[*] ?"`